tests/integration/test_credential_secret_view.py

```diff
@@ -101,21 +92,21 @@ def test_update(self, client, logged_in_user, team, credential, secret):
 
     def test_update_nonexistent_team(self, client, logged_in_user, team, credential, secret):
         unknown_pui = '00000000-0000-0000-0000-000000000000'
-        url = f'/team/{unknown_pui}/credential/{credential.public_identifier}/secrets/{secret.key}/'
+        url = f'/team/{unknown_pui}/credentials/{credential.public_identifier}/secrets/{secret.key}/'
```

Comment:
I think this test needs an update: change the url, move the team-id to the input data

Reply:
Actually, because this is a credential secret endpoint, it has no link to the team (only indirectly via the credential itself), so this test can be removed.